spec/asyncapi.md

```diff
@@ -273,6 +273,7 @@ This object MAY be extended with [Specification Extensions](#specificationExtens
 
 ##### Info Object Example
 
+<!-- asyncapi-example-tester:{"name":"Info Object","json_path":"$.info"} -->
```

Comment:
How are we testing yaml version of the examples? You didn't add comment annotations

Reply:
I initially started with json examples, so I removed the yaml comment. Will add it for testing also...

Reply:
As we discussed in the past, you can transform those yaml to json on runtime and then validate as you do for json

Reply:
> As we discussed in the past, you can transform those yaml to json on runtime and then validate as you do for json. Just a suggestion.

Yes, I'll exactly do that. That's the approach I had in mind. Thanks for the reminder!

Reply:
resolved via 53290ea

Reply:
added yaml testing through cf62169